src/services/donations/donations.hooks.js

```diff
@@ -401,11 +401,15 @@ const populateSchema = () => context => {
 
   return context;
 };
-
+const aggregate = async context => {
+  if ('_aggregate' in context.params.query) {
+    context.result = context.service.Model.aggregate(context.params.query._aggregate);
+  }
+};
 module.exports = {
   before: {
     all: [commons.paramsFromClient('schema')],
-    find: [sanitizeAddress('giverAddress')],
+    find: [sanitizeAddress('giverAddress'), aggregate],
```

Comment:
Does it bypass donations service find process? I mean it's a `before hook`, maybe after this the donations service core functionality is called and adds some data. I am thinking about implementing this as a separate service. Like this one: https://github.com/feathersjs-ecosystem/feathers-mongodb/issues/114#issuecomment-635597269